libdnf5-cli/output/pkg_colorizer.cpp

```diff
@@ -106,4 +107,30 @@ std::string PkgColorizer::to_escape(const std::string & color) {
     return output;
 }
 
+std::string PkgColorizer::get_coloring_description() {
+    std::ostringstream desc;
+    desc << "(";
+
+    // [NOTE](mfocko) Is there a possibility of any other meaning?
```

Comment:
Even though PkgColorizer is only used with these meanings, we should keep it generic. Maybe `get_coloring_description` could take in the descriptions as 4 strings, similar to the constructor? Or we could add a new constructor and store the descriptions as member variables.